config/routes.rb

```diff
@@ -1,4 +1,55 @@
 Rails.application.routes.draw do
+
+  # because we need to create a session immediately upon entering the site, for guest users
+  root 'sessions#create_order'
+
+  resources :products do
+    resources :reviews, :only => [:new, :create]
+  end
```

Comment:
As a user, I can't seem to add a new product. I get a heroku error. Not sure if this was working locally or if it's one of those things heroku picks up on. I see it's passing in the id of the user as a query string after products/new. Any ideas why this is not working?